vxquery-xtest/src/test/resources/Queries/XQuery/Indexing/useIndex7.xq

```diff
@@ -18,8 +18,8 @@
 (: Search Lucene Index :)
 (: Find all the weather readings for Washington state for a specific day    :)
 (: 2002-2-2.                                                                  :)
-for $s in collection-from-index("target/tmp/indexFolder", "/stationCollection/station")/station
-for $r in collection-from-index("target/tmp/indexFolder", "/dataCollection/data")/data
+for $s in collection-from-index("src/test/resources/TestSources/ghcnd", "/stationCollection/station")/station
```

Comment:
Can you use the sources key similar to the GHCND queries instead of the file path? You can see the map of sources in the top of the VXQueryCatalog.xml file.